### PR TITLE
Add sound when opening granary info window.

### PR DESCRIPTION
**Pull request by wonderwagon**
- Add all walker sounds to array
- Convert tabs to spaces in building_type.h, building_sounds.js, walker_sounds.js
- Align text within walker and building sound arrays

### CODE REVIEW

File: src/building/building_type.h [hunk 1]
```diff
@@ -3,263 +3,263 @@
...
-	BUILDING_MENU_ENTERTAINMENT = 286,
-	BUILDING_MENU_ADMINISTRATION = 287,
-	BUILDING_MENU_SECURITY = 288,
-	BUILDING_MENU_INDUSTRY = 289,
-	BUILDING_MENU_GUILDS = 290,
-	BUILDING_MENU_LARGE_TEMPLES = 291,
-	BUILDING_MENU_FOOD = 292,
-	BUILDING_MENU_DISTRIBUTION = 293,
-	BUILDING_MENU_DEFENCES = 294,
+    BUILDING_NONE = 0,
```
Comment (dalerank) [inline src/building/building_type.h:6]: please remove those changes, because it non depends to this commit

File: src/window/window_granary_info.cpp [hunk 2]
```diff
@@ -7,6 +7,7 @@
 #include "city/warnings.h"
 #include "window/building/common.h"
 #include "window/window_building_info.h"
+#include "sound/sound_building.h"
```
Comment (dalerank) [inline src/window/window_granary_info.cpp:10]: why this need?

File: src/scripts/building_sounds.js [hunk 3]
```diff
@@ -1,86 +1,89 @@
...
-	{ type: BUILDING_HOUSE_ELEGANT_RESIDENCE,	sound: "Wavs/housing_r.wav"},
-	{ type: BUILDING_HOUSE_FANCY_RESIDENCE,	sound: "Wavs/housing_r.wav"},
-	{ type: BUILDING_HOUSE_COMMON_MANOR,	sound: "Wavs/housing_r.wav"},
-	{ type: BUILDING_HOUSE_SPACIOUS_MANOR,	sound: "Wavs/housing_r.wav"},
-	{ type: BUILDING_HOUSE_ELEGANT_MANOR,	sound: "Wavs/housing_r.wav"},
-	{ type: BUILDING_HOUSE_STATELY_MANOR,	sound: "Wavs/housing_r.wav"},
-	{ type: BUILDING_HOUSE_MODEST_ESTATE,	sound: "Wavs/housing_r.wav"},
-	{ type: BUILDING_HOUSE_PALATIAL_ESTATE,	sound: "Wavs/housing_r.wav"},
-	{ type: BUILDING_WATER_SUPPLY,	sound: "Wavs/fountain.wav"},
+  { type: BUILDING_TAX_COLLECTOR,            sound: "Wavs/TAXFARM.wav"},
```
Comment (dalerank) [inline src/scripts/building_sounds.js:4]: leave only meaning changes, pls
if you want make codestyle changes move it to different commit

**Review by dalerank — CHANGES_REQUESTED** [review]

File: src/building/building_type.h (hunk 1 above)
Comment (wonderwagon) [reply src/building/building_type.h]: Style changes moved, can be in a later pull request

File: src/window/window_granary_info.cpp (hunk 2 above)
Comment (wonderwagon) [reply src/window/window_granary_info.cpp]: Removed unneeded header include

File: src/scripts/building_sounds.js (hunk 3 above)
Comment (wonderwagon) [reply src/scripts/building_sounds.js]: Codestyle changes removed.
